sympy/polys/polytools.py

```diff
@@ -2669,6 +2669,124 @@ def subresultants(f, g):
 
         return list(map(per, result))
 
+
+    def subresultant_polys(f, g):
```

Comment:
It should be one blank line between methods like the methods above.

Reply:
Done